spec/requests/api/v1/projects/enabled_dynamic_fields_spec.rb

```diff
@@ -0,0 +1,151 @@
+require 'rails_helper'
+
+RSpec.describe 'Enabled Dynamic Fields Requests', type: :request do
```

Comment:
At some point we'll need to add include_examples 'requires user to have correct permissions' do and check for project admin permissions, but that can come later.  Probably want to add a TODO though.

Reply:
see other comment